packages/amplify-e2e-tests/src/__tests__/smoke-tests/smoketest.test.ts

```diff
@@ -224,11 +199,11 @@ function writeBanner(text: string) {
   process.stdout.write('\n');
 }
 
-describe('Release Smoke Tests', () => {
-  const createCommands = (amplify: Amplify, cliVersion: string, directory: string): Command[] => [
+describe('Smoke Tests', () => {
+  const createCommands = (amplify: Amplify, directory: string): Command[] => [
     {
-      description: `Install @aws-amplify/cli@${cliVersion}`,
-      run: () => NPM.install(`@aws-amplify/cli@${cliVersion}`, true),
+      description: 'Amplify CLI version',
+      run: () => amplify.version(),
     },
```

Comment:
I moved CLI installation to Actions logic, so that we can reuse same smoke test with many installation strategies.